src/main/kotlin/string/addition/caculator/domain/Operand.kt

```diff
@@ -0,0 +1,21 @@
+package string.addition.caculator.domain
+
+data class Operand(private val inputStr: String) {
+    var number: Int
+        private set
```

Comment:
```suggestion
    val number: Int
```
number를 변경하는 곳이 없어서 value로 선언해도 될 것 같아요.